kotlinpoet/src/main/java/com/squareup/kotlinpoet/Util.kt

```diff
@@ -15,6 +15,7 @@
  */
 package com.squareup.kotlinpoet
 
+import java.lang.IllegalArgumentException
```

Comment:
I think we should throw `kotlin.IllegalArgumentException` instead. Just removing the import should do the job.